assets/samplesheet.csv

```diff
@@ -1,3 +1,3 @@
 id,fasta,reference,structures
-seatoxin-ref,/home/luisasantus/Desktop/crg_cluster/projects/nf-core-msa/test-dataset/setoxin-ref.fa,/home/luisasantus/Desktop/crg_cluster/projects/nf-core-msa/test-dataset/setoxin.ref,/home/luisasantus/Desktop/crg_cluster/projects/nf-core-msa/test-dataset/structures/setoxin-ref
-toxin-ref,/home/luisasantus/Desktop/crg_cluster/projects/nf-core-msa/test-dataset/toxin-ref.fa,/home/luisasantus/Desktop/crg_cluster/projects/nf-core-msa/test-dataset/toxin.ref,
+seatoxin-ref,../test-dataset/setoxin-ref.fa,/home/luisasantus/Desktop/crg_cluster/projects/multiplesequencealign/test-dataset/setoxin.ref,/home/luisasantus/Desktop/crg_cluster/projects/multiplesequencealign/test-dataset/structures/setoxin-ref
+toxin-ref,../test-dataset/toxin-ref.fa,/home/luisasantus/Desktop/crg_cluster/projects/multiplesequencealign/test-dataset/toxin.ref,
```

Comment:
you still have some local paths leftovers

Reply:
oh ok, yes i have those files in the repo. I imagine this is not supposed to happen then? Should i add it to the nf-core test datasets instead?